reposilite-backend/src/main/kotlin/com/reposilite/token/api/AccessToken.kt

```diff
@@ -68,9 +68,16 @@ enum class AccessTokenPermission(val identifier: String, val shortcut: String) {
         fun findAccessTokenPermissionByIdentifier(identifier: String): AccessTokenPermission =
             values().first { it.identifier == identifier }
 
+        private fun findAccessTokenPermissionByIdentifierNullable(identifier: String): AccessTokenPermission? =
```

Comment:
Might be a good idea to just improve these old methods rather than introducing the new one with `Nullable` suffix. In sources that required non-null values just use `!!` / provide an error